selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/RenameWorkspaceTest.java

```diff
@@ -28,8 +28,8 @@
 
 /** @author Andrey Chizhikov */
 public class RenameWorkspaceTest {
-  private static final String MIN_WORKSPACE_NAME = NameGenerator.generate("", 3);
-  private static final String MAX_WORKSPACE_NAME = NameGenerator.generate("", 100);
+  private static final String MIN_WORKSPACE_NAME_SIZE = generate("", 3);
```

Comment:
I meant naming of "magic numbers" 3 and 100 https://refactoring.guru/replace-magic-number-with-symbolic-constant